packages/account/src/Styles/account.scss

```diff
@@ -1458,6 +1458,11 @@ $MIN_HEIGHT_FLOATING: calc(
 .onfido-container {
     margin-top: 24px;
 
+    .ods-button {
```

Comment:
ods 🤔 ?

Reply:
new class name from Onfido 😆 but i added a more specific selector as there was a request to override certain styles 🙏🏼